### PR TITLE
Wrist Mk2.1 info integration

### PR DESCRIPTION
**Pull request by Lawproto**
Adding some info regarding our latest designed wrist, also moving some bits from the hand section to the wrist.

### CODE REVIEW

File: docs/wrists/wrist_mk2.md [hunk 1]
```diff
@@ -1,112 +1,36 @@
-# Wrist Mk2: the Mathematics of the spherical joint
-
+# Wrist Mk2.0 vs Wrist Mk2.1
 <p align="center">
-  <img  src=    "../img/wrist_mk2.png"
-        title=  "wrist_mk2.png"
-        width=  "750">
+  <img  src=    "./img/mk2.0_vs_mk2.1.png"
```
Comment (Lawproto) [inline docs/wrists/wrist_mk2.md:3]:
```suggestion
  <img  src=    "../img/mk2.0_vs_mk2.1.png"
```

File: docs/wrists/wrist_mk2.md [hunk 2]
```diff
@@ -1,112 +1,36 @@
...
-
-The total torque applied to the platform is:
-
-$$
-\tau=q_1\times F_1 +q_2\times F_2 +q_3\times F_3.
-$$
-
-We also assume that the momentum of inertia of the platform is negligible with respect to the viscous friction, so that we have a first order law of motion $\omega=k\tau$. In this way, given the position of the arms, by iterating until convergence we arrive in the correspondant platform orientation by small finite rotations starting from a tentative initial configuration.
+<p align="center">
+  <img  src=    "./img/pitches.png"
```
Comment (Lawproto) [inline docs/wrists/wrist_mk2.md:21]:
```suggestion
  <img  src=    "../img/pitches.png"
```

File: docs/wrists/wrist_mk2.md [hunk 3]
```diff
@@ -1,112 +1,36 @@
...
+<p align="center">
+  <sub>Wrist Mk2 and Elbow joints 1/2</sub>
+</p>
 
-- At runtime, we use as initial configuration the orientation obtained in the previous control cycle, so that we are sure to be starting from a configuration close to the real one.
-- If no variations occured in the encoder readings, there is no need for any calculation. 
-- For better numeric stability it is better to represent internally the platform orientation with a quaternion, normalizing it periodically.
\ No newline at end of file
+<p align="center">
+  <img  src=    "./img/wrist-yaw-roll.png"
```
Comment (Lawproto) [inline docs/wrists/wrist_mk2.md:30]:
```suggestion
  <img  src=    "../img/wrist-yaw-roll.png"
```

File: docs/wrists/wrist_mk2_math.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+- The coordinates of the arm joints, that rotates around the vertical axis, are $p_1=(\cos{\theta_1},\sin{\theta_1},-h)$, $p_2=(\cos{\theta_2},\sin{\theta_2},-h)$ and $p_3=(\cos{\theta_3},\sin{\theta_3},-h)$ with $h$ constant and with $\theta_i$ = rotations of the three motors. In the case of symmetrical structure, at rest we have $\theta_1=90^\circ$, $\theta_2=210^\circ$ and $\theta_3=330^\circ$.
+- The above numbers are referred to the **left** employment of the wrist mechanism. In order to use it in the **right** configuration, an offset of $180^\circ$ must be added so that the whole mechanism kinematics will result symmetrically reflected, maintaining the meaning of *roll, pitch and yaw* under the chosen convention. Thus, at rest, the arm angles will be $\theta_1=270^\circ$, $\theta_2=30^\circ$ and $\theta_3=150^\circ$. This implies that the rest position must be set by properly calibrating the arm position sensors on the software side, while the hand interface must be rotated by $180^\circ$ with respect to the platform on the hardware side.
```
Comment (pattacini) [inline docs/wrists/wrist_mk2_math.md:22]: >Thus, at rest, the arm angles will be $\theta_1=270^\circ$, $\theta_2=30^\circ$ and $\theta_3=150^\circ$. This implies that the rest position must be set by properly calibrating the arm position sensors on the software side, while the hand interface must be rotated by $180^\circ$ with respect to the platform on the hardware side.

Is this still valid for Mk2.1?

If not, I would update this section to be compatible with the latest version only.

**Review by pattacini — CHANGES_REQUESTED** [review]
Hi @Lawproto 

Thanks for the fixes!

We also need to fix/update the following in the main navigation section:

https://github.com/Lawproto/documentation/blob/wrist_patch/mkdocs.yml#L169-L170

File: docs/wrists/wrist_mk2_math.md (hunk 4 above)
Comment (Lawproto) [reply docs/wrists/wrist_mk2_math.md:22]: good point! I think not, but I have to read more in depth and understand what those angles refer to.

**Comment by pattacini** [comment]
Thanks heaps @Lawproto for the updates!

The last point to address is:

>We also need to fix/update the following in the main navigation section:
>
>https://github.com/Lawproto/documentation/blob/wrist_patch/mkdocs.yml#L169-L170

File: docs/wrists/wrist_mk2_math.md (hunk 4 above)
Comment (Lawproto) [reply docs/wrists/wrist_mk2_math.md]: @pattacini I corrected the angles to the current situation. Fun fact: Mk2.0 was not symmetrical as the text stated but Mk2.1 is instead and made it all work again.

**Comment by Lawproto** [comment]
> Thanks heaps @Lawproto for the updates!
> 
> The last point to address is:
> 
> > We also need to fix/update the following in the main navigation section:
> > https://github.com/Lawproto/documentation/blob/wrist_patch/mkdocs.yml#L169-L170

@pattacini done.

**Review by pattacini — APPROVED** [review]

**Pull request merged**